### PR TITLE
Fix moduleAttributesVersion errors with stage-0 preset in babel standalone

### LINKED ISSUE #11628: babel stand alone version 7.10.0 doesnt work
7.10 doent seems do do anything at all. null behavoir. I had to revert to 7.9

### PR DESCRIPTION
**Pull request by hamlim**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #11628
| Patch: Bug Fix?          | ✅ 
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

Changes:

* Expose `moduleAttributesVersion` config option on the `stage-0` preset for babel standalone
* Forward that `moduleAttributesVersion` onto the config for `stage-1` preset
* Fix version mismatch in the `module-attributes` plugin and `plugin-utils` file (one was expecting `version: 'apr-2020'` and the other was expecting `version: 'may-2020'`, I **assumed** that may is the correct version here) See: https://github.com/babel/babel/blob/master/packages/babel-parser/src/plugin-utils.js#L95-L101
* Updated config in the `example.htm` file for testing the changes locally

Testing steps:

* `make prepublish-build-standalone`
* `npx serve ./packages/babel-standalone`
* Visit localhost and navigate to `examples/example.htm`
* Verify that babel **doesn't** hit an error when transforming the code from the textarea

### CODE REVIEW

File: packages/babel-plugin-syntax-module-attributes/src/index.js [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { declare } from "@babel/helper-plugin-utils";
 export default declare((api, { version }) => {
   api.assertVersion(7);
 
-  if (typeof version !== "string" || version !== "apr-2020") {
+  if (typeof version !== "string" || version !== "may-2020") {
```
Comment (nicolo-ribaudo) [inline packages/babel-plugin-syntax-module-attributes/src/index.js:6]: Good catch!

File: packages/babel-standalone/examples/example.htm [hunk 2]
```diff
@@ -1,38 +1,49 @@
 <!DOCTYPE html>
 <html>
```
Comment (nicolo-ribaudo) [inline packages/babel-standalone/examples/example.htm:2]: Could you revert the changes to this file?

**Comment by nicolo-ribaudo** [comment]
I now we don't have many tests for `standalone`, but could you add one to https://github.com/babel/babel/blob/master/packages/babel-standalone/test/babel.js? 🙏

**Comment by hamlim** [comment]
> I now we don't have many tests for `standalone`, but could you add one to [/packages/babel-standalone/test/babel.js@`master`](https://github.com/babel/babel/blob/master/packages/babel-standalone/test/babel.js) [](/babel/babel/blob/HEAD@{2020-05-27T17:28:57Z}/packages/babel-standalone/test/babel.js)? 🙏

Sure! Would something like asserting that stage-0 doesn't throw when passing `moduleAttributesVersion` work?

**Comment by hamlim** [comment]
Added a small regression test 👍

File: packages/babel-standalone/examples/example.htm [hunk 3]
```diff
@@ -1,38 +1,49 @@
...
+      function transform() {
+        try {
+          outputEl.innerHTML = Babel.transform(inputEl.value, {
+            presets: [
+              "es2015",
+              "react",
+              [
+                "stage-0",
+                {
+                  decoratorsBeforeExport: false,
```
Comment (JLHwung) [inline packages/babel-standalone/examples/example.htm:34]: I don't think we need to change the example once we offers default value to these options. The example should serve as a minimal working config.

**Pull request merged**